board/arm-64bit-toolchain.cmake

```diff
@@ -0,0 +1,55 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR ARM)
+
+if(MINGW OR CYGWIN OR WIN32)
+    set(UTIL_SEARCH_CMD where)
+elseif(UNIX OR APPLE)
+    set(UTIL_SEARCH_CMD which)
+endif()
+
+## You need to add the full path in TOOLCHAIN_PREFIX, if you do not use the
+## standard gcc toolchain 
+set(TOOLCHAIN_PREFIX aarch64-linux-gnu-)
+
+## In some distribution, a dynamic link for aarch64-linux-gnu-gcc may not be
```

Comment:
Just out of curiosity, is that naming convention (`aarch64-linux-gnu-gcc`) standard across distributions?

Reply:
It should be the same, this should be different only when using a different cross-compiler toolchain.
The main issue sometimes is the missing symbolic link toward the `aarch64-linux-gnu-gcc`, therefore I have added the option to add the version at the end. For instance, you might find `aarch64-linux-gnu-gcc-5` instead of `aarch64-linux-gnu-gcc`